pom.xml

```diff
@@ -670,6 +670,63 @@
 				</dependencies>
 			</dependencyManagement>
 		</profile>
+		<profile>
+			<id>zookeeper-discovery</id>
+			<properties>
+				<apache.curator.version>4.2.0</apache.curator.version>
+				<apache.zookeeper.version>3.4.14</apache.zookeeper.version>
+				<zookeeper.discovery.version>3.0.4</zookeeper.discovery.version>
+			</properties>
+			<dependencyManagement>
```

Comment:
It looks like this dependencyManagement section is not necessary as the zookeeper related dependencies are managed in [spring-cloud-zookeeper-dependencies](https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-zookeeper-dependencies/3.0.4) which is imported in [spring-cloud-dependencies](https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-dependencies/2020.0.4).